protoc_plugin/test/map_field_test.dart

```diff
@@ -267,4 +267,39 @@ void main() {
     testMap.mergeFromBuffer(testMap2.writeToBuffer());
     expect(testMap.int32ToMessageField[2].secondValue, 42);
   });
+
+  test('Freeze message with map field', () {
+    TestMap testMap = TestMap();
+    _setValues(testMap);
+    testMap.freeze();
+
+    try {
```

Comment:
You can write these tests of throwing a message better with:
```
expect(() => doStuffTheShouldThrow(), throwsA(const TypeMatcher<UnsupportedError>()))
```

Reply:
Done